### PR TITLE
Add table configuration class

### PR DESCRIPTION
**Pull request by m-rudyk**
# Description

Add an option to configure database tables for the open-source LPVS.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [x] Test Coverage update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Below is just an example and you can delete it when you contribute.

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by tdrozdovsky — APPROVED** [review]
LGTM

**Review by o-kopysov — APPROVED** [review]
Do we need to add app.table.* into application.properties?

**Comment by m-rudyk** [comment]
> Do we need to add app.table.* into application.properties?

Yes, sure. We will do that in configuration PR (there are 2 more steps required).

File: src/main/java/com/lpvs/util/LPVSTableConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LPVSTableConfiguration {
+    @Value("${app.table.detectedLicenseName}")
+    private String detectedLicenseName;
+
+    @Value("${app.table.detectedLicenseShema}")
+    private String detectedLicenseShema;
```
Comment (tiokim) [inline src/main/java/com/lpvs/util/LPVSTableConfiguration.java:19]: Isn't it `schema`, not shema?

**Pull request merged**